tests/cypress/integration/projects.spec.js

```diff
@@ -1,8 +1,52 @@
-/// <reference types="Cypress" />
+import { generate_random_string } from '../support/commands.js'
 
-context('Homepage', () => {
-  it('should have the congrats message', () => {
-    cy.visit('http://34.248.52.210:8000/')
-    cy.get('main h2').should('have.text', 'The install worked successfully! Congratulations!')
+context('Projects', () => {
+  var random_string
```

Comment:
Have you try using `let`? If we can use `let` it will be clearer for the scope of a variable.

Reply:
@gatukgl Thanks!